spec/factories/user.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :user do
+    password              '123'
+    email                 'user@example.com'
+    password_confirmation '123'
+    name                  'gerard'
+    user_name             'grdm'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.